tests/bayesian/test_apical_tiebreak_bayesian_temporal_memory.py

```diff
@@ -3,10 +3,7 @@
 import numpy as np
 
 # Import apical tiebreak bayesian tm
-import sys
-import os
-sys.path.append(os.path.realpath(os.path.join(os.getcwd(), '../algorithms')))
-import apical_tiebreak_bayesian_temporal_memory as btm
+import htmresearch.algorithms.apical_tiebreak_bayesian_temporal_memory as btm
```

Comment:
I'd prefer the previous implementation since it is necessary to re-deploy the whole project whenever you want to test it with the most recent changes if you want to import it that way. If you don't like the first one, is there another workaround?
If we took the previous, we'd need to adapt the path of course tho since it is in another directory now

Reply:
Solved this after peer discussion